src/pages/Form/FormsSkeleton.tsx

```diff
@@ -0,0 +1,31 @@
+import { Card, CardContent, CardHeader, Skeleton, Stack } from "@mui/material";
+
+function FormsSkeleton() {
+  return (
+    <Card sx={{ blockSize: "100%" }} className="mt-10">
+      <CardHeader
+        title={
+          <Stack
+            direction="row"
+            justifyContent="space-between"
+            alignItems="center"
+          >
+            <Skeleton variant="text" width="50%" />
+            <Skeleton variant="text" width="10%" />
+          </Stack>
+        }
+      />
+      <CardContent>
+        <Stack direction="column" gap={4}>
+          <Skeleton variant="rounded" height={130} />
```

Comment:
Why not stay consistent and use tailwind?

Reply:
It's because those material ui components don't have classname, it requires writing pure css in `sx` prop